bpdm-gate/src/main/kotlin/org/eclipse/tractusx/bpdm/gate/service/BusinessPartnerService.kt

```diff
@@ -110,7 +112,9 @@ class BusinessPartnerService(
         val externalIds = entityCandidates.map { it.externalId }
         assertInputStageExists(externalIds)
 
-        val resolutionResults = resolveCandidatesForStage(entityCandidates, StageType.Output)
+        val validatedEntities = filterUpdateCandidates(entityCandidates, StageType.Output)
```

Comment:
I find it a little inconsistent that you are using one `filterUpdateCandidates` in `upsertBusinessPartnersInput` but the other in `upsertBusinessPartnersOutputFromCandidates`. Any reasons for that?

Reply:
This question still seems to be open..

Reply:
Yeah, basically in the upsertBusinessPartnersOutput is currently not being used, because the finishCleaningTask maps directly to a BusinessPartner and not to BusinessPartnerOutputRequest which is used in this method. This was done because otherwise two mappings we're needed. Probably some refactoring later will need to be done.